run-test-docker-image.sh

```diff
@@ -22,17 +22,14 @@ check_test_execution() {
 }
 
 
-############ Setup docker and git environments
-docker system prune --force
-git submodule update --init --recursive
-
-
 ############ Parse optional parameters
 # One option to the test script is '-s' which gives the option to skip building the image
 skip_build=''
-while getopts 's' flag; do
+wait=300 # 5 minutes; wait is used for sleep while the container is setting up Volttron
+while getopts 'sw:' flag; do
   case "$flag" in
     s) skip_build=true ;;
+    w) wait="$OPTARG" ;;
     *) echo "Unexpected option ${flag}"
```

Comment:
add the group (i.e. your username) and tag to the arguments (i.e. develop) then from your github action you can pass those in (perhaps default to VOLTTRON and :develop).

Reply:
Good point. Allows users to custom name their local images if they want to differentiate and for testing.